.github/workflows/canary.yml

```diff
@@ -0,0 +1,60 @@
+name: Pre-release
+
+on:
+  push:
+    branches:
+      - next
+      - snapshot-releases
+
+jobs:
+  release:
+    # Prevents changesets action from creating a PR on forks
+    if: github.repository == 'modernweb-dev/web'
+    name: Pre-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
Is this needed?

Reply:
Looks like it defaults to GPR (maybe). I grabbed this from https://github.com/modernweb-dev/web/blob/master/.github/workflows/release.yml#L26 originally...